src/job.ts

```diff
@@ -46,6 +46,8 @@ export type JobOptions = JobRequest<JobMetadata>;
 export type CancelCallback = RequestCallback<bigquery.IJobCancelResponse>;
 export type CancelResponse = [bigquery.IJobCancelResponse];
 
+export type DeleteCallback = RequestCallback<object>;
```

Comment:
`<object>` is super generic - what actually gets returned by the API?

Reply:
Maybe we should be extending this type?  https://github.com/googleapis/nodejs-common/blob/b1ceb532ac7bf4852f430c83af5a71618ed42f54/src/service-object.ts#L119